pwndbg/prompt.py

```diff
@@ -5,17 +5,38 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import os
+import subprocess
+
 import gdb
 
 import pwndbg.events
 import pwndbg.memoize
 import pwndbg.stdio
 
-msg = "Loaded %i commands.  Type pwndbg [filter] for a list." % len(pwndbg.commands._Command.commands)
-print(pwndbg.color.red(msg))
 
+def show_version():
```

Comment:
We should likely do something similar to Pwntools and have a `version.py` which has the release version: https://github.com/Gallopsled/pwntools/blob/dev/pwnlib/version.py